### PR TITLE
[MINOR] Updating website as latest release supports Scala 2.12

### PR DESCRIPTION
**Pull request by pavansai1997**
Latest Release supports Scala 2.12

### CODE REVIEW

**Comment by pavansai1997** [comment]
cc : @ajbozarth , @lmccay , @dacort  Can you please review and merge the PR.

File: site/get-started.md [hunk 1]
```diff
@@ -31,7 +31,7 @@ Download Livy packages from [here]({{ site.baseurl }}/download).
 
 ### 2. Run Livy
 To run the Livy server, you will also need an Apache Spark installation. You can get Spark releases at
-<https://spark.apache.org/downloads.html>. Livy requires at least Spark 1.6 and supports both Scala 2.10 and 2.11
+<https://spark.apache.org/downloads.html>. Livy requires at least Spark 1.6 and supports Scala 2.10, 2.11 and 2.12
```
Comment (dacort) [inline site/get-started.md]:
```suggestion
<https://spark.apache.org/downloads.html>. Livy requires at least Spark 1.6 and supports both Scala 2.11 and 2.12
```

Don't believe we build for 2.10 anymore.

**Comment by pavansai1997** [comment]
@dacort made necessary changes, can you review and merge

**Comment by pavansai1997** [comment]
@dacort made necessary changes, can you review and merge

**Review by lmccay — APPROVED** [review]
LGTM
+1

**Comment by lmccay** [comment]
@pavansai1997 - thanks for your contribution!

**Comment by dacort** [comment]
Thanks @lmccay - I also pushed the change to https://github.com/apache/incubator-livy-website/tree/asf-site so it's live now.

**Pull request merged**